.github/workflows/release.yaml

```diff
@@ -0,0 +1,42 @@
+name: Create and publish a container image
+
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
If we use commits explicitly, I guess we should change this as well? Also, I'd use `v2-0-gec3a7ce`, so we have both tag annotation and commit reference in a single revision.